src/main/java/com/gachigang/ontherun/service/AuthService.java

```diff
@@ -36,12 +55,75 @@ public void authenticateUser(LoginRequest loginRequest) {
         SecurityContextHolder.getContext().setAuthentication(authentication);
     }
 
-    public User register(RegisterRequest registerRequest) {
+    public AuthenticationResponse register(RegisterRequest registerRequest) {
```

Comment:
Bad idea to return DTO from service layer, we decided to return entities to make this layer and methods re-usable.